elasticmagic/compiler.py

```diff
@@ -811,6 +812,13 @@ def visit_search_query_context(self, query_ctx):
             query_ctx.track_total_hits is not None
         ):
             params['track_total_hits'] = query_ctx.track_total_hits
+
+        if (
+            self.features.supports_search_after and
```

Comment:
I don't think it is a good idea to silently ignore user expectations on results of his/her query. It would be better to return an error in such a case. Either we can throw an exception or Elasticsearch will fail. I prefer to allow Elasticsearch to decide on its own.

_Features_ was made to bypass some differences in Elasticsearch API which we are able to handle. We cannot cover all those differences.